src/Nethermind/Nethermind.AuRa/Contracts/SystemContract.cs

```diff
@@ -54,9 +54,23 @@ protected Transaction GenerateTransaction(byte[] transactionData, long gasLimit
         
         public void InvokeTransaction(BlockHeader header, ITransactionProcessor transactionProcessor, Transaction transaction, CallOutputTracer tracer)
         {
-            if (transaction != null && !TryInvokeTransaction(header, transactionProcessor, transaction, tracer))
+            if (transaction != null)
```

Comment:
why would the transaction ever be null in such a call?

Reply:
Not really, unless dev error. Will remove.